bridges/snowbridge/primitives/inbound-queue/src/v2/converter.rs

```diff
@@ -55,6 +57,8 @@ pub struct MessageToXcm<
 	GatewayProxyAddress,
 	EthereumUniversalLocation,
 	GlobalAssetHubLocation,
+	AssetHubUniversalLocation,
+	AccountId,
```

Comment:
Seems unused since you're configuring `ExternalConsensusLocationsConverterFor` to directly convert to `[u8; 32]`.

```suggestion
```

Reply:
I meant to use it, added in https://github.com/claravanstaden/polkadot-sdk/pull/19/commits/fe67940d4133aba0625e54b2eea4a3cffe23d929.